main.py

```diff
@@ -1,13 +1,102 @@
 #!/usr/bin/env python
+import re
+from nltk.wsd import lesk
 
-# Your classes or methods go here:
 
+class JsonParsedWriter:
 
-class Tool:
-    def my_method(self):
-        print("my_method called!")
+    @staticmethod
+    def write(data, output_name, indent=2):
+        import json
+        with open(output_name, 'w') as out_file:
+            parsed = json.loads(json.dumps(data))
+            json.dump(parsed, out_file, indent=indent)
+
+
+class TweetCleaner:
+
+    def __init__(self, input_file_name):
+
+        self.name = input_file_name
+
+        # list of patterns for regular expressions
+        self.tags_patt = re.compile(r",\[.*\]")
```

Comment:
Yes, this is a good idea to pre-create all the data you need in your methods right here, in your `__init__()`.